test/cypress/e2e/oidc/oidc_auth_test.spec.js

```diff
@@ -63,7 +63,7 @@ describe('Log in via OIDC', () => {
 
     cy.visit('http://localhost:5601/app/dev_tools#/console');
 
-    cy.get('a').contains('Dev Tools').should('be.visible');
+    cy.get('a[data-test-subj="breadcrumb first last"]').contains('Dev Tools').should('be.visible');
```

Comment:
Are there multiple anchors now?

Reply:
Yes the PR introduced multiple anchors. In any case this is a better practice since without this it will break with any introduction.